dscreator/datasets/timeseries/glomma.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from dataclasses import dataclass
+
+import xarray as xr
+from dataclasses import asdict
+
+from dscreator import utils
+from dscreator.cfarray.attributes import VariableAttrs, DatasetAttrsDiscrete
+from dscreator.datasets.base import TimeseriesDatasetBuilder
+
+
+@dataclass
+class BaterodBuilder(TimeseriesDatasetBuilder):
+    def dataset_attributes(self, ds: xr.Dataset) -> DatasetAttrsDiscrete:
+        """Add ACDD attributes to a xarray dataset
+
+        Add attributes following the Attribute Convention for Data Discovery to a dataset, also see https://adc.met.no/node/96.
+        A good keywords viewer is located here https://gcmd.earthdata.nasa.gov/KeywordViewer.
+        For converting CF standard names erddap also contains a good converter for cf standard names.
+        """
+        return DatasetAttrsDiscrete(
+            title="Sensor station at Baterød in Glomma",
+            summary="Sensor measurements collected by a sensor station at Baterød in Glomma.",
+            title_no="Sensorstasjon ved Baterød i nedre Glomma",
+            summary_no="Sensor målinger samlet i Glomma ved Baterød vannverk. Hensikten med stasjonen er å overvåke naturlig dynamikk og trender i viktige fysisk-kjemiske vannkvalitetsparametere.",
+            keywords=",".join(
+                [
+                    # GEMET & NORTHEMES
+                    "GCMDSK:EARTH SCIENCE > TERRESTRIAL HYDROSPHERE > SURFACE WATER > SURFACE WATER FEATURES > RIVERS/STREAMS",
+                    "GCMDLOC:CONTINENT > EUROPE > NORTHERN EUROPE > SCANDINAVIA > NORWAY",
+                    "GEMET:Hydrography",
+                    "NORTHEMES:Weather and climate",
+                ]
+            ),
+            keywords_vocabulary=",".join(
+                [
+                    "GCMDSK:GCMD Science Keywords:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/sciencekeywords",
+                    "GCMDPLA:GCMD Platforms:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/platforms",
+                    "GCMDLOC:GCMD Locations:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/locations",
+                    "GEMET:INSPIRE Themes:http://inspire.ec.europa.eu/theme",
+                    "NORTHEMES:GeoNorge Themes:https://register.geonorge.no/metadata-kodelister/nasjonal-temainndeling",
+                ]
+            ),
+            iso_topic_category="inlandWaters",
+            featureType=ds.attrs["featureType"],
+            date_created=utils.iso_now(),
+            project="AquaINFRA",
```

Comment:
Shouldn't it be the "Elveovervåkingsprogrammet"? We are only sharing the data with AquaINFRA but they come from/belong to the Elveovervåkingsprogrammet

Reply:
or should it be both?

Reply:
We should also add the `processing_level="Experimental"` here

Reply:
> or should it be both?

Hmm, not sure, we're also not adding AquaINFRA to Ferrybox data, or are we?

Reply:
we do